src/plugins/intel_cpu/src/nodes/inverse.cpp

```diff
@@ -153,7 +151,7 @@ void Inverse::lu_decomposition(const T* data,
 
         // Find maximum value pivot - non-parallel
         for (size_t i = (k + 1) * m_side, j = k + 1; i < m_side_squared; i += m_side, ++j) {
-            if (std::abs(U[i + k]) > std::abs(U[pivot_idx + k])) {
+            if (abs(U[i + k]) > abs(U[pivot_idx + k])) {
```

Comment:
```suggestion
            if (std::abs(U[i + k]) > std::abs(U[pivot_idx + k])) {
```
Use std version.
There will be compilation warning which can be tread as an error